docs/system-properties.adoc

```diff
@@ -13,7 +13,8 @@ For example, clearing a system property for a test execution can be done as foll
 @Test
 @ClearSystemProperty(key = "some property")
 void test() {
-	assertThat(System.getProperty("some property")).isNull();
+	assertThat(System.getProperty("some property")).
+			isNull();
```

Comment:
Other examples are only inteded once, this uses two tabs?

Reply:
When introducing a linebreak, but the assertion is not completed the "broke" part should be intended one tab more than the regular lines. In this example the `isNull` should be intended more than the `assertThat` as it belongs to the further line.

Reply:
Hmmm, how is that different from e.g. this line:

https://github.com/junit-pioneer/junit-pioneer/pull/318/files#diff-1d7cb2850ba029a8ff112abb5d62575aR30

Here and elsewhere one tab is used, whereas the above example uses two tabs. So:

```java
assertThat(/* ... */).
		isNull();

// vs.

asserThat(/* ... */).
	isEqualTo(/* ... */);
```

Reply:
Good spot 👍 

Should be correct now. I also formated the code samples in retrying-failed-tests to match the same as in the other docs.